docs/charts/bar.md

```diff
@@ -85,6 +86,7 @@ the color of the bars is generally set this way.
 | Name | Description
 | ---- | ----
 | `label` | The label for the dataset which appears in the legend and tooltips.
+| `order` | The drawing order of dataset. Also affects stacking order.
```

Comment:
Maybe instead of "Also affects stacking order" it should say "Also affects order for stacking, tooltip, and legend"